### DISCUSSION TITLE
usePopover - prevent popover getting closed on blur

### DISCUSSION THREAD

**Discussion by samithaf** [adobe/react-spectrum#5341] [Q&A] [answered]
Hello, 

Is there a way to prevent popover getting closed on blur and only can be closed using a close button?

Is there a prop such as `shouldCloseOnBlur`. 

E.g. popover

![Screenshot 2023-11-01 at 9 44 35 pm](https://github.com/adobe/react-spectrum/assets/1860417/826ea957-872b-4bfc-a9c7-f0c632fab6e5)

**Comment by reidbarber** [comment, marked as answer]
[useOverlay](https://react-spectrum.adobe.com/react-aria/useOverlay.html) has `shouldCloseOnInteractOutside`, and we are considering adding that to usePopover and passing it through to useOverlay as a part of #4976. Besides that, perhaps you could make the isOpen state controlled, and only close it when the button is pressed.
